metrics/identical_codeblocks.py

```diff
@@ -35,8 +34,8 @@ def identical_blocks_of_code(self, files) -> int | float:
         et = parse(StringIO(res.stdout))
         for child in et.getroot():
             if child.tag == "duplication":
-                count += 1
-        return count
+                data.append(["Identical Code", "Find way to get file", 1, 1])
```

Comment:
Maybe CPD has something built in so you can get the filepath as well? The XML nodes seem to have a memory address built in: <Element 'duplication' at 0x125cabab0>

Reply:
Will look into it